Tools/TestWebKitAPI/Tests/WebCore/ImageBufferTests.cpp

```diff
@@ -336,4 +393,11 @@ INSTANTIATE_TEST_SUITE_P(ImageBufferTests,
         testing::Values(TestImageBufferOptions::NoOptions, TestImageBufferOptions::Accelerated)),
     TestParametersToStringFormatter());
 
+INSTANTIATE_TEST_SUITE_P(ImageBufferTests,
+    AnyTwoImageBufferOptionsTest,
+    testing::Combine(
+        testing::Values(TestImageBufferOptions::NoOptions, TestImageBufferOptions::Accelerated),
+        testing::Values(TestImageBufferOptions::NoOptions, TestImageBufferOptions::Accelerated)),
```

Comment:
Did you mean to pass the same `TestImageBufferOptions::Accelerated` for the two tests?

Reply:
No, combine is a cartesian product.
`Combine({a,b}, {a,b})` means `{a,b} x {a,b}`, e.g it generates list `{aa, ab, ba, bb}`. 
So this is what we want: test all the possible combinations of source and destination.